h2o-genmodel/src/main/java/hex/genmodel/algos/glrm/GlrmLoss.java

```diff
@@ -158,16 +158,29 @@
     @Override public double mloss(double[] u, int a) {
       if (!(a >= 0 && a < u.length))
         throw new IndexOutOfBoundsException("a must be between 0 and " + (u.length - 1));
```

Comment:
this check can be removed because you're doing the same check in the other `mloss()`